distributed/nanny.py

```diff
@@ -144,7 +144,7 @@ def __init__(
 
         self.Worker = Worker if worker_class is None else worker_class
         self.env = env or {}
-        self.config = config or {}
+        self.config = config or dask.config.config
```

Comment:
Will create another PR for this. This prohibits setting worker configs in the tests

- [ ] PR for Nanny/config

Reply:
https://github.com/dask/distributed/pull/4378